app/models/piece.js

```diff
@@ -24,18 +19,14 @@ export default Model.extend({
     inverse: null,
   }),
 
-  documentContainer: belongsTo('document', {
+  documentContainer: belongsTo('document-container', {
     inverse: null,
   }),
-  document: deprecatingAlias('documentContainer', {
-    id: 'model-refactor.documents',
-    until: '?',
-  }),
-  nextVersion: belongsTo('document-version', {
-    inverse: 'previousVersion',
+  nextPiece: belongsTo('piece', {
```

Comment:
Hier vind ik `nextVersion` (`previous...`) toch duidelijker. Een `piece` heeft een relatie `nextVersion` en die is ook van type `piece`. `nextPiece` klinkt meer alsof het eender wat kan zijn, niet?

Reply:
Ik vind persoonlijk van niet. 
Next piece zegt voor mij precies wat het volgende gaat zijn (nl een piece)
Waar een version kan gebruikt worden door verschillende componenten. of anders
(piece) -> nextpiece = piece
of
(piece) -> nextversion = piece